.kitchen.yml

```diff
@@ -17,13 +17,13 @@ platforms:
 - name: amazon-linux
   driver_config:
     box: mvbcoding/awslinux
+- name: amazonlinux-2
+  driver_config:
+    box: ywatase/amzn2
```

Comment:
I like this, however I wonder if we should build our own regularly with boxcutter/bento?

Reply:
Sure, could you point me in the right direction on this? Maybe open an issue and I'll tackle in a separate PR / could update it then in other cookbooks that are using amazon.

Reply:
https://github.com/chef/bento

Reply:
And here's the box we need to copy https://github.com/ywatase/packer-amazon-linux2

Reply:
(I'd like to be using sous-chefs for both tbh)

Reply:
Tim has a PR in flight for this it seems https://github.com/chef/bento/pull/1144